hack/arktos-cni.rc

```diff
@@ -33,7 +33,7 @@ setup_cni_conf() {
 
     echo "copying ${conf_source_file} to ${conf_target_file}..."
     local cni_conf_dir=$(dirname ${conf_target_file})
-    local backup_dir="$(dirname ${cni_conf_dir})/$(basename ${cni_conf_dir})_"$(date -d "today" +"%Y%m%d%H%M")
+    local backup_dir="/var/backups/$(basename ${cni_conf_dir})_"$(date -d "today" +"%Y%m%d%H%M")
```

Comment:
what is the reason that this changed to hardcode?

Reply:
The backup dir should be different from coni conf dir. But /var/backup should be replaced by local variable.